src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -1682,5 +1686,227 @@ internal static ulong ConvertToUInt64Checked(double value)
             ThrowHelper.ThrowOverflowException();
             return 0;
         }
+
+        [StackTraceHidden]
+        internal static int DivideChecked(int dividend, int divisor)
```

Comment:
Naming nit: There are no checked vs. unchecked variants of div and mod. I would drop the checked suffix.